Documentation/ceph-csi-snapshot.md

```diff
@@ -6,15 +6,24 @@ indent: true
 
 ## Prerequisites
 
-* Requires Kubernetes v1.17+ which supports snapshot beta.
+### Snapshot Support Matrix
 
-* Install the new snapshot controller and snapshot beta CRD. More info can be found
-[here](https://github.com/kubernetes-csi/external-snapshotter/tree/master#usage)
+| Snapshot API version | Kubernetes Version    | Snapshot-Controller Version         | Sidecar Version                          |
```

Comment:
Does the user really need this full table? If the snapshot controller v4 works for both v1 or v1beta1 snapshots, seems like we could keep it simple and just recommend the v4 controller for 1.17 or newer.

Reply:
And we can say we just officially support 1.17 or newer for snapshots. Maybe we don't even need to mention the alpha snapshotter.